ci/test_python_singlegpu.sh

```diff
@@ -31,16 +31,20 @@ rapids-logger "pytest cuml single GPU"
   --cov-report=xml:"${RAPIDS_COVERAGE_DIR}/cuml-accel-coverage.xml" \
   --cov-report=term
 
-rapids-logger "memory leak pytests"
 
-./ci/run_cuml_singlegpu_memleak_pytests.sh \
-  --numprocesses=1 \
-  --junitxml="${RAPIDS_TESTS_DIR}/junit-cuml-memleak.xml" \
-  --cov-config=../../.coveragerc \
-  --cov=cuml \
-  --cov-report=xml:"${RAPIDS_COVERAGE_DIR}/cuml-memleak-coverage.xml" \
-  --cov-report=term \
-  -m "memleak"
+if [ "${RAPIDS_BUILD_TYPE}" == "nightly" ]; then
+  rapids-logger "memory leak pytests"
+
+  ./ci/run_cuml_singlegpu_memleak_pytests.sh \
+    --numprocesses=1 \
+    --junitxml="${RAPIDS_TESTS_DIR}/junit-cuml-memleak.xml" \
+    --cov-config=../../.coveragerc \
+    --cov=cuml \
+    --cov-report=xml:"${RAPIDS_COVERAGE_DIR}/cuml-memleak-coverage.xml" \
+    --cov-report=term \
```

Comment:
Slightly off topic: one thing that is a bit frustrating is the amount of output in our test runs, especially because we run several pytest commands per job. Is there anyone who looks at the coverage report in the terminal? I think I basically never look at it, ever. So maybe we can ditch this line (on all `pytest` invocations)?

Reply:
Agreed, but let's tackle this on a follow up PR